src/System.Threading/tests/System.Threading.Tests.builds

```diff
@@ -9,7 +9,7 @@
     </Project>
     <Project Include="System.Threading.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>net46</TestTFMs>
+      <TestTFMs>net463</TestTFMs>
```

Comment:
What does `<TargetGroup>netstandard1.3</TargetGroup>` do here? I see a few test project .builds files that don't have this, but have an OSGroup instead.